src/common/BrowserConnector.ts

```diff
@@ -14,14 +14,13 @@
  * limitations under the License.
  */
 
+import Config from './Config.js';
```

Comment:
I really like this PR, thank you! 

How do you feel about changing `Config` to `Environment` ? I think that might read slightly nicer.